doc/user_guide/electron_holography.rst

```diff
@@ -57,5 +58,38 @@ This option can be used for batch processing as the reconstruction_param can be
 
     >>> wave_image1 = im1.reconstruct_phase(rec_param=reconstruction_param)
 
-Further analyses of the reconstructed wave (phase) can be done using the :py:class:`~._signals.wave_image.ComplexSignal2D` class
-functionality (see dedicated documentation).
\ No newline at end of file
+Further analyses of the reconstructed wave (phase) can be done using :py:class:`~._signals.wave_image.ElectronWaveImage` class
+functionality (see bellow).
+
+
+The ElectronWaveImage class
+===========================
+
+The :py:class:`~._signals.wave_image.ElectronWaveImage` class can hold information about the complex electron
+wave. As such, relevant properties like the `amplitude`, `phase` and the `real` and `imag` part can be
+directly accessed and return appropriate :py:class:`~._signals.signal2d.Signal2D` signals.
+
+To transform a :py:class:`~._signals.complex_signal2d.ComplexSignal2D` (or subclass) into a
+:py:class:`~._signals.wave_image.ElectronWaveImage` use:
+
+.. code-block:: python
+
+    >>> im.set_signal_type('ElectronWaveImage')
```

Comment:
We can't break the API in 1.1, only in 2.0. We have to be strict with this, as failing to do so would undermine the confidence of users and developers who may based their packages on HyperSpy. For an example, see @vidartf's HyperSpyUI. Currently it is broken due to all the API changes. Maintaining it would be a nightmare if we cannot guarantee that the API will remain untouched until HyperSpy 2.0.

Of course we could take the approach that you propose and add a `_legacy_signal_type` list to allow to make these changes without breaking the API. +1 for this!

Regarding the docs, now that @sem-geologist has fixed the automatic read the docs built, the docs are sync with master in [RtD](http://hyperspy.readthedocs.io/en/latest/). If you want to compile them locally you can use sphinx. This is trivial in linux (simply executing `make html` in `hyperspy/doc` does it), but I've been told that it may not be so straightforward in Windows.